docs/ABOUT.md

```diff
@@ -1,8 +1,16 @@
 # About
 
-Lisp Flavoured Erlang (AKA LFE) is a lisp syntax front-end to the Erlang compiler. 
+**Lisp Flavoured Erlang** (AKA LFE) is a lisp syntax **front-end** to the **Erlang compiler**. 
 Code produced with it is compatible with "normal" Erlang code. 
 LFE is a (proper) Lisp based on the features and limitations of the Erlang VM.
+LFE is useful in applications that require the **robustness** and **concurrency** handling of Erlang, combined with the **flexibility** of a Lisp-family language.
+
+**Key Benefits:**
```

Comment:
We list out key features for LFE at https://github.com/exercism/lfe/blob/13ec9033ee17a056d7f9c2acb6e8fc7af813fa2b/config.json#L498. This appears on the track's about page at https://exercism.org/tracks/lfe/about underneath the contents of this file. There we shouldn't repeat that information here.